mappings/net/minecraft/client/gui/widget/option/KeyBindListWidget.mapping

```diff
@@ -10,14 +10,14 @@ CLASS net/minecraft/unmapped/C_aarntxpx net/minecraft/client/gui/widget/option/K
 		METHOD <init> (Lnet/minecraft/unmapped/C_aarntxpx;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
 			ARG 2 title
 	CLASS C_pdmfuuxr Entry
-	CLASS C_rlmrzmiu KeyBindingEntry
+	CLASS C_rlmrzmiu KeyBindEntry
 		FIELD f_amogjyre bindButton Lnet/minecraft/unmapped/C_buwziidm;
-		FIELD f_ibgocqec bindingName Lnet/minecraft/unmapped/C_rdaqiwdt;
+		FIELD f_ibgocqec keyName Lnet/minecraft/unmapped/C_rdaqiwdt;
 		FIELD f_isjsjuxy resetButton Lnet/minecraft/unmapped/C_buwziidm;
-		FIELD f_myzuvtet binding Lnet/minecraft/unmapped/C_sygikzty;
-		METHOD <init> (Lnet/minecraft/unmapped/C_aarntxpx;Lnet/minecraft/unmapped/C_sygikzty;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
-			ARG 2 binding
-			ARG 3 bindingName
+		FIELD f_myzuvtet key Lnet/minecraft/unmapped/C_sygikzty;
+		METHOD <init> KeyBindEntry (Lnet/minecraft/unmapped/C_aarntxpx;Lnet/minecraft/unmapped/C_sygikzty;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
```

Comment:
```suggestion
		METHOD <init> (Lnet/minecraft/unmapped/C_aarntxpx;Lnet/minecraft/unmapped/C_sygikzty;Lnet/minecraft/unmapped/C_rdaqiwdt;)V
```

Reply:
Oh, I guess Enigma did that automatically